src/neptune/attributes/series/fetchable_series.py

```diff
@@ -35,20 +35,24 @@
 
 Row = TypeVar("Row", StringSeriesValues, FloatSeriesValues)
 
+MAX_FETCH_LIMIT = 1000
+
 
 class FetchableSeries(Generic[Row]):
     @abc.abstractmethod
     def _fetch_values_from_backend(self, offset, limit) -> Row:
         pass
 
-    def fetch_values(self, *, include_timestamp: bool = True, progress_bar: Optional[ProgressBarType] = None):
+    def fetch_values(
+        self,
+        *,
+        include_timestamp: bool = True,
+        progress_bar: Optional[ProgressBarType] = None,
+        offset: Optional[int] = None,
+        limit: Optional[int] = None,
```

Comment:
We might want to add some value checks, because when I tried with a negative limit I got
```
neptune.common.exceptions.ClientHttpError: 

----ClientHttpError-----------------------------------------------------------------------

The Neptune server returned the status 400.

The server response was:
{"headers":{},"code":400,"errorType":"BAD_REQUEST","title":"Limit must not be negative"}
```

which is probably not the most user-friendly. (Similar case is for the `offset` param)